index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" data-theme="lemonade">
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/daisyui@4.11.1/dist/full.min.css" rel="stylesheet" type="text/css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+    <h1 class="hero text-3xl font-bold bg-base-200">
+        <div class="hero-content flex-col lg:flex-row" style="gap:0rem">
+            <img src="./icons/teapot.svg" class="size-14"/>Tea Time Game
+        </div>
+    </h1>
+    <div class="hero bg-base-200" style="min-height:60vh">
```

Comment:
![image](https://github.com/danipurdes/playteatime/assets/3577548/092bf39d-173d-4403-a092-68f664c77a97)

Could we make the min-height a bit smaller to reduce the gap between the header and the hero section?

Reply:
This screenshot was taken in Edge on PC, but it looked very similar in Chrome on PC as well

Reply:
Wow, yeah, that looks huge compared to mine

Reply:
Fixed in latest push!